Gruntfile.js

```diff
@@ -45,15 +45,18 @@ module.exports = function (grunt) {
     parser.parseString(xmlBuffer, function (err, result) {
       // Limit cities data to Dutch name and zip code. That's all we currently
       // need in the application.
-      cities = result.cdbxml.cities.city.map(
-        function (city) {
-          return {
-            'label': city.zip + ' ' + city.labelnl,
-            'name': city.labelnl,
-            'zip': city.zip
-          };
-        }
-      );
+      cities = result.cdbxml.cities.city.reduce(
+        function (filteredCities, city) {
+          if (city.submunicipality && city.submunicipality === 'true') {
+            filteredCities.push({
+              'label': city.zip + ' ' + city.labelnl,
+              'name': city.labelnl,
+              'zip': city.zip
+            });
+            return filteredCities;
+          }
+          return filteredCities;
+        }, []);
```

Comment:
Can't this be written much simpeler.

Something like:

`cities = _.filter(result.cdbxml.cities.city, (city) => (typeof city.submunicipality === 'boolean'))
`

Reply:
I'm using the lodash filter function here because I don't think you can use it on Array itsef in this version of javascript

Reply:
We lose the mapping in that case. But it could be rewritten to first filter() and then map(). Not sure if we can use the short syntax you're using in this app though.

Reply:
Yes indeed I would opt for a map and filter for better readability

Reply:
I will write it with two method calls, first `filter` then `map`.

But I think `typeof city.submunicipality === 'boolean'` is maybe wrong because it's stored as a string inside the XML